src/app/beer_garden/systems.py

```diff
@@ -163,10 +163,8 @@ def upsert(system: System) -> System:
         The created / updated system
     """
     try:
-        return create_system(system)
+        return create_system(system, _publish_error=False)
```

Comment:
This is new to me. I did not know that you could pass in undefined variables like this. Everything works, just wanted to let you know that I didn't know about this.

Reply:
It's not really undefined, it's just intended for the decorator "wrapper" function instead of the normal "wrapped" one. That's why I use `pop`:

```
 _publish_success = kwargs.pop("_publish_success", True)
```

 That way those kwargs are removed before the wrapped function call.